tutorials/colors/colorbar_only.py

```diff
@@ -0,0 +1,108 @@
+"""
+=============================
+Customized Colorbars Tutorial
+=============================
+
+This tutorial shows how to build colorbars without an attached plot.
+
+"""
+
+###############################################################################
```

Comment:
not a big deal but you technically don't need to be commenting out lines here, just extend the `"""` up until the first bit of code